apps/mark-scan/backend/src/app.ts

```diff
@@ -195,6 +200,12 @@ export function buildApi(
     setAcceptingPaperState(): void {
       assert(stateMachine);
       stateMachine.setAcceptingPaper();
+
+      // TODO(kofi): Remove once we've added mock paper handler functionality to the
+      // dev dock:
+      if (isMockPaperHandler(paperHandler)) {
```

Comment:
This gets moved client-side in commit https://github.com/votingworks/vxsuite/pull/4979/commits/5de0640c2ddb661b4ab753ee4fe515b353bb7174 - sorry meant to rebase and clean up this commit